.github/issue-close-require.yml

```diff
@@ -0,0 +1,23 @@
+name: Issue Close Require
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+permissions:
+  contents: read
+
+jobs:
+  issue-close-require:
+    permissions:
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: need reproduce
+        uses: actions-cool/issues-helper@v3
+        with:
+          actions: "close-issues"
+          labels: "need reproduction"
+          inactive-day: 3
+          body: |
+            Since the issue was labeled with `need reproduction`, but no response in 3 days. This issue will be closed. If you have any questions, you can comment and reply.
```

Comment:
```suggestion
            Since the issue was labeled with `need reproduction`, but no response in 3 days. This issue will be closed. Feel free to comment and reopen it if you have any further questions.
```